.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,31 @@
+name: Build and Push Docker Image to ACR
+
+on:
+  release:
+    types: [published]
+    branches:
+      - main
```

Comment:
Should it use the tagged branch?

Reply:
it will monitor whether there is a new release made for main branch.